org.testeditor.web.dropwizard.xtext/src/test/java/org/testeditor/web/xtext/index/persistence/GitServiceInitTest.xtend

```diff
@@ -2,20 +2,40 @@ package org.testeditor.web.xtext.index.persistence
 
 import java.io.File
 import org.eclipse.jgit.api.Git
+import org.eclipse.jgit.api.errors.TransportException
 import org.eclipse.jgit.lib.Constants
 import org.eclipse.jgit.revwalk.RevCommit
 import org.eclipse.jgit.transport.URIish
+import org.junit.Rule
 import org.junit.Test
+import org.junit.rules.ExpectedException
+import org.junit.rules.TemporaryFolder
 
 class GitServiceInitTest extends AbstractGitTest {
+	
+	@Rule public TemporaryFolder keyfiles = new TemporaryFolder
+	@Rule public ExpectedException expectedException = ExpectedException.none
+	
+	@Test
+	def void cloneTriesToUsePrivateKeyIfConfigured() {
```

Comment:
… the name threw me off a bit: the method actually tests whether a particular exception is thrown if the private key is invalid.

How about another test with a valid private key?